rocky/katalogus/templates/partials/plugin_tile.html

```diff
@@ -30,18 +30,34 @@
         {% endif %}
         <span class="plugin-author-link de-emphasized"><a target="_blank" href="https://openkat.nl" rel="noopener noreferrer">OpenKAT</a></span>
         {% if show_report_types %}
-            <div class="tags">
-                <p>
-                    <strong>{% translate "Required for:" %}</strong>
-                </p>
-                {% for plugin_id, report_types in plugin_report_types.items %}
-                    {% if plugin_id == plugin.id %}
-                        {% for report_type in report_types %}
-                            <span class="label tags-color-{{ report_type.label_style }}">{{ report_type.name }}</span>
-                        {% endfor %}
+            {% for plugin_id, report_types in plugin_report_types.items %}
+                {% if plugin_id == plugin.id %}
+                    {% if report_types|length >= 2 %}
+                        <div class="dropdown">
+                            <button aria-controls="plugin-report-types"
+                                    aria-expanded="false"
+                                    class="dropdown-button ghost">
+                                {% translate "Required for" %} {{ report_types|length }} {% translate "report types" %}
+                                <span aria-hidden="true" class="icon ti-chevron-right"></span>
+                            </button>
+                            <div class="dropdown-list" id="plugin-report-types">
+                                <ul>
+                                    {% for report_type in report_types %}<li>{{ report_type.name }}</li>{% endfor %}
+                                </ul>
+                            </div>
```

Comment:
Should be a modal / dialog element.

Reply:
The dropdown implementation currently follows something resembling the APG [Menu Button Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/). Are you sure it should be a Dialog instead? If so, why?

Reply:
@underdarknl  In the design, it is opening a modal, but I don't see why we need to open a dialog where there is no dialog. It just shows a list op report types. There is no user interaction. Maybe it is designed that way because we are going to expand this feature to show more info or to really invoke a dialog?

Reply:
Manon has no dialogs yet. A dialog does usually serve the purpose of asking user input (even an alert dialog ask the user to acknowledge the alert), but if this was designed to look and function like a modal dialog, then it makes sense to use KAT's existing modal dialog component. :+1: 

Whether that design is "correct" is another matter. I don't really see a reason why this should be a modal element.

Oh, by the way, if you'll allow a bit of pedantry: **"modality"** is a property of a UI element - any element, really - meaning that it blocks the user from interacting with any other part of the UI. A dialog _can_ be modal (and often is) but doesn't _need_ to be. It could just as well allow interaction with other parts of the UI while staying open.